tests/sentry/utils/performance_issues/test_consecutive_db_detector.py

```diff
@@ -297,25 +260,10 @@ def test_respects_project_option(self):
         assert not detector.is_creation_allowed_for_project(project)
 
     def test_detects_consecutive_db_does_not_detect_php(self):
-        span_duration = 50
-        spans = [
-            create_span(
-                "db",
-                span_duration,
-                "SELECT `customer`.`id` FROM `customers` WHERE `customer`.`name` = $1",
-            ),
-            create_span(
-                "db",
-                span_duration,
-                "SELECT `order`.`id` FROM `books_author` WHERE `author`.`type` = $1",
-            ),
-            create_span("db", 900, "SELECT COUNT(*) FROM `products`"),
-        ]
-        spans = [
-            modify_span_start(span, span_duration * spans.index(span)) for span in spans
-        ]  # ensure spans don't overlap
+        event = self.create_issue_event()
+
+        assert len(self.find_problems(event)) == 1
```

Comment:
@gggritso fyi, this should resolve an issue that you mentioned in an earlier PR, where its possible the event doesn't have the issue in the first place when checking if PHP detects an issue.